src/audio/chain_dma.c

```diff
@@ -18,13 +21,21 @@ DECLARE_SOF_RT_UUID("chain_dma", chain_dma_uuid, 0x6a0a274f, 0x27cc, 0x4afb,
 		    0xa3, 0xe7, 0x34, 0x44, 0x72, 0x3f, 0x43, 0x2e);
 DECLARE_TR_CTX(chain_dma_tr, SOF_UUID(chain_dma_uuid), LOG_LEVEL_INFO);
 
-static struct comp_dev *chain_dma_create(const struct comp_driver *drv,
+static struct comp_dev *chain_task_create(const struct comp_driver *drv,
```

Comment:
Hrm, you are fixing what you have added in the first patch? Why not do it properly from the start?

Reply:
iow, squash the patches.